yt_dlp/extractor/youtube.py

```diff
@@ -3135,6 +3135,9 @@ def _extract_player_url(self, *ytcfgs, webpage=None):
             get_all=False, expected_type=str)
         if not player_url:
             return
+        # TODO: Add proper support for the 'tce' variant players
+        # See https://github.com/yt-dlp/yt-dlp/issues/12398
+        player_url = player_url.replace('/player_ias_tce.vflset/', '/player_ias.vflset/')
```

Comment:
Can we add a warning or debug message if a tce player is received? In the case down the line it has a different nsig calculation which would cause 403s